### PR TITLE
feat(error-messages): Update error messages in preview

### PR DESCRIPTION
**Pull request by marcoartaviaq**
Update preview error messages to let users know they can download/open the file to view it.

### CODE REVIEW

File: src/i18n/en-US.properties [hunk 1]
```diff
@@ -67,29 +67,31 @@ file_preview_label=File preview
...
 # Default preview error message
-error_unsupported=We’re sorry, we can’t preview this file type in your web browser. To view this content please download and open it on your device.
+error_unsupported=File type unsupported for preview. Open or download the file to view.
 # Account doesn’t have a sufficient tariff to preview the requested file type
-error_account=We’re sorry, your account is unable to preview this file type.
+error_account=We’re sorry, your account is unable to preview this file type. Open or download the file to view.
 # No permissions preview error message
 error_permissions=We’re sorry, you don’t have permission to preview this file.
-# Preview refresh error message suggesting refreshing the page as a possible fix
-error_refresh=We’re sorry, the preview didn’t load. Please refresh the page.
```
Comment (jstoffan) [inline src/i18n/en-US.properties:78]: The messages seem very inconsistent. Some have conciliatory language (e.g. "We're sorry") and some don't. Some have personalized language (e.g. "Your browser") and some don't. We should standardize the form and tone holistically before merging.

Comment (marcoartaviaq) [reply src/i18n/en-US.properties:78]: Hi @jstoffan,

I’ve updated the messages with input from the product team to ensure they are more standardized.

File: src/i18n/en-US.properties [hunk 2]
```diff
@@ -67,35 +67,35 @@ file_preview_label=File preview
...
 # Preview error message shown when conversion was unable to process the file at the given time.
-error_try_again_later=We’re sorry the preview didn’t load. Please try again later.
-# Preview error message shown when conversion failed due to file contents
-error_bad_file=We’re sorry the preview didn’t load. This file could not be converted.
+error_try_again_later=This preview didn’t load. Please refresh the page or download file to view.
+# Preview error message shown when conversion failed due to file contents, suggesting opening or downloading as a possible fix
+error_bad_file=This preview didn’t load. Please try to open or download the file to view.
 # Preview error message shown when the file cannot be downloaded
-error_not_downloadable=Oops! It looks like something is wrong with this file. We apologize for the inconvenience and recommend that you upload a new version of this file or roll back to a previous version. Please contact us for more details.
+error_not_downloadable=Something seems to be wrong with this file. Please refresh the page, and if the issue continues try to upload a new version.
```
Comment (jstoffan) [inline src/i18n/en-US.properties:94]:
```suggestion
error_not_downloadable=Something seems to be wrong with this file. Please refresh the page, and if the issue continues, try to upload a new version.
```

**Review by jstoffan — DISMISSED** [review]
LGTM, thanks for refining the messages!

**Review by jstoffan — APPROVED** [review]

**Review by jstoffan — APPROVED** [review]

**Pull request merged**